sym_api_client_python/processors/sym_message_parser.py

```diff
@@ -60,6 +60,14 @@ def get_im_name(self, message_data):
     def get_stream_id(self, message_data):
         return message_data['stream']['streamId']
 
+    def get_tags(self, json_nodes, tag_type):
```

Comment:
I would make this method private `def _get_tags(self, json_nodes, tag_type):` like what has been done in Java sdk: [getTags](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/blob/6fd04cd6f09ffa10d8926667cfe350e570bd6bc0/symphony-bdk-legacy/symphony-api-client-java/src/main/java/utils/SymMessageParser.java#L72)

Reply:
done.